.github/workflows/ci.yml

```diff
@@ -14,7 +14,13 @@ jobs:
           - "6.7.0"
           - "7.0.0"
           - "7.8.2"
+        terraform-version:
+          - "1.2.9"
     steps:
+      # avoid confusion around which terraform is being used
+      - name: rm existing terraform
```

Comment:
I'm puzzled as to how there comes to be an existing terraform binary there, but I don't wanna hold up your fix so 🚀